packages/material-ui-lab/src/Autocomplete/Autocomplete.test.js

```diff
@@ -429,6 +429,24 @@ describe('<Autocomplete />', () => {
       fireEvent.keyDown(firstSelectedValue, { key: 'ArrowRight' });
       expect(textbox).toHaveFocus();
     });
+
+    it('should not be required if a value is selected', () => {
```

Comment:
I don't understand how this relates to the issue. Whether a field is required is not tied to the current state of a field. It  seems like validation logic is broken which affects way more than `required`. We should revert this change and fix the underlying issue instead. Otherwise we're building technical debt.

Why didn't we add a test that replayed the original issue?

Reply:
What alternative fix do you have in mind?

Reply:
I don't know about the fix. I just know that this is not fixing the issue but the symptoms while also creating new issues.

Reply:
It's definitely taking a shortcut. I have tried to explore the alternatives, couldn't find anything relevant. I would be in favor of:

- Writing a more high-level test, requiring #21691
- Waiting for users to report these "news issues", I don't see what you are referring to. My hope is that it's worth the tradeoff.